stock_quant_package_fast_move/models/stock_quant_package.py

```diff
@@ -89,9 +91,10 @@ def _move_to_location(self, location, destination_package=None):
             package_level = self.env["stock.package_level"].create(package_level_vals)
             package_level.write({"is_done": True})
 
-        # Validate the picking
         picking.action_confirm()
-        picking.button_validate()
+        if no_validate:
```

Comment:
Should't it be `if not no_validate` ?

`If you check 'Don't Validate', the created internal picking won't be validated. Otherwise, it will be validated and the package will directly moved to the specified destination location`